server/server.js

```diff
@@ -93,19 +106,13 @@ app.post(`${apiBaseUrl}/employees`,
 
 // Get user roles
 app.get(`${apiBaseUrl}/roles`,
-  keycloak.protect((token, req) => {
-    const roles = [];
-    ['admin', 'maximus'].forEach((item) => {
-      if (token.hasRole(item)) {
-        roles.push(item);
-      }
-    });
-    req.roles = roles;
-    return !token.isExpired();
-  }),
+  keycloak.protect(),
   asyncMiddleware(async (req, res) => {
+    const { roles } = req.kauth
+      .grant.access_token.content
```

Comment:
@fwkendall `req.kauth.grant.access_token.content` contains a lot of info including the user first and last name. I just didn’t figure out yet how to get values from the Attributes, I created a token mapper but seems not working.